Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml

```diff
@@ -0,0 +1,46 @@
+# TestGrid Test Configuration Parameters
+version: '0.9'
+emailToList: "bfsi-group@wso2.com,divya@wso2.com,malshani@wso2.com,verushka@wso2.com,charithad@wso2.com,akilaa@wso2.com,rivindu@wso2.com"
+infrastructureConfig:
+  iacProvider: CLOUDFORMATION
+  infrastructureProvider: AWS
+  containerOrchestrationEngine: None
+  includes:
+    - Ubuntu-18.04
+    - SQLServer-SE-13.00
+    - OPEN_JDK8
+  provisioners:
+    - name: distributed-single-node-deployment
+      remoteRepository: "git@github.com:wso2-support/aws-open-banking.git"
+      remoteBranch: "support-2.0.0"
+      description: Provide Infra for running scenario tests
+      scripts:
+        - name: 'prod-wso2ob-200-deployment'
+          description: ''
+          type: CLOUDFORMATION
+          file: obam-with-obkm-au.yaml
```

Comment:
change the name of the file to the correct name

Reply:
correction done [655a665](https://github.com/wso2/testgrid-job-configs/pull/559/commits/655a665f56e6d68e92b908014b08dddbd1fd0aff)